src/main/java/com/lina/airline/utils/ValidationUtils.java

```diff
@@ -8,4 +8,6 @@ public final class ValidationUtils {
     public static final String NAME_VALIDATION = "^[A-Za-z\\s'-]+$\n";
     public static final String EMAIL_VALIDATION ="^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,6}$";
     public static final String PASSWORD_VALIDATION ="^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,30}$";
+    public static final   String PHONE_VALIDATION = "^\\+?[0-9. ()-]{10,15}$";
```

Comment:
Phone number validation should cover others more condition also. 

## Phone Number Validation Requirements:
- Required: The phone number should not be empty.

- E.164 Format:

  - This format includes:
  - A plus sign (+).
  -Country code (1 to 3 digits).
  -Subscriber number (up to 15 digits).
- No spaces, dashes, or parentheses allowed. Only digits and the leading + symbol are allowed.
Example: +14155552671 (for US number)

- Length:

  - Total length should be at most 15 digits (excluding the plus sign).
  - Country codes are typically 1-3 digits, and local subscriber numbers vary by country.
- Start with "+":

  - Ensure the phone number starts with a + followed by the country code.
Digits Only After "+":

 - After the +, the phone number must consist of digits only.
- Optional Validation:

  - You can implement library-based validation (such as Google's [libphonenumber](https://github.com/google/libphonenumber)) to ensure the phone number is valid according to the country code and format.
  - The library can also detect invalid country codes and format discrepancies for different regions.